drivers/adc/adc_ads1x1x.c

```diff
@@ -20,6 +21,16 @@
 
 LOG_MODULE_REGISTER(ADS1X1X, CONFIG_ADC_LOG_LEVEL);
 
+#if DT_ANY_COMPAT_HAS_PROP_STATUS_OKAY(ti_ads1115, alert_rdy_gpios) || \
+    DT_ANY_COMPAT_HAS_PROP_STATUS_OKAY(ti_ads1114, alert_rdy_gpios) || \
+    DT_ANY_COMPAT_HAS_PROP_STATUS_OKAY(ti_ads1113, alert_rdy_gpios) || \
+    DT_ANY_COMPAT_HAS_PROP_STATUS_OKAY(ti_ads1014, alert_rdy_gpios) || \
+    DT_ANY_COMPAT_HAS_PROP_STATUS_OKAY(ti_ads1013, alert_rdy_gpios)
```

Comment:
What is the recommendation here?
I think it looks ugly if they are not aligned, hehehe

Reply:
I think it looks fine